app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -0,0 +1,64 @@
+module Idv
+  class HowToVerifyController < ApplicationController
+    include IdvStepConcern
+    include RenderConditionConcern
+
+    before_action :confirm_step_allowed
+
+    check_or_render_not_found -> { self.class.enabled? }
+
+    def show
+      analytics.idv_doc_auth_how_to_verify_visited(**analytics_arguments)
+      @idv_how_to_verify_form = Idv::HowToVerifyForm.new
+    end
+
+    def update
+      result = Idv::HowToVerifyForm.new.submit(how_to_verify_form_params)
+
+      analytics.idv_doc_auth_how_to_verify_submitted(
+        **analytics_arguments.merge(result.to_h),
+      )
+
+      if result.success?
+        if how_to_verify_form_params['selection'] == Idv::HowToVerifyForm::REMOTE
+          redirect_to idv_hybrid_handoff_url
+        else
+          redirect_to idv_document_capture_url
+        end
+      else
+        flash[:error] = result.first_error_message
+        redirect_to idv_how_to_verify_url
+      end
+    end
+
+    def self.step_info
+      Idv::StepInfo.new(
+        key: :how_to_verify,
+        controller: controller_name,
+        next_steps: [:hybrid_handoff, :document_capture],
+        preconditions: ->(idv_session:, user:) do
+          self.enabled?
+        end,
+      )
+    end
+
+    def self.enabled?
+      IdentityConfig.store.in_person_proofing_opt_in_enabled
+    end
+
+    private
+
+    def analytics_arguments
+      {
+        step: 'how_to_verify',
+        analytics_id: 'Doc Auth',
+      }
```

Comment:
There are missing analytic args.  Here is a good example [here](https://github.com/18F/identity-idp/blob/cf88ff0bbaeb2209296e1463176d9eab311947ed/app/controllers/idv/phone_question_controller.rb#L47C6-L55).